src/cls/IPM/Storage/UpdateStep.cls

```diff
@@ -39,6 +39,30 @@ Method %OnNew(
     return $$$OK
 }
 
+/// Returns %IPM.Storage.UpdateStep object corresponding to unique index in storage table
+/// If object does not exist, this will return an empty object
+ClassMethod GetUpdateStepObject(
+	className As %String,
+	methodName As %String,
+	mirrored As %Boolean,
```

Comment:
Doesn't look like mirrored is used so need not be here. Also, can call the vars hostName and instanceName since they don't have anything to do with mirroring (same as the properties in the table)